packages/commonwealth/client/styles/modals/archive_thread_modal.scss

```diff
@@ -0,0 +1,37 @@
+@import '../shared';
+
+.ArchiveThreadModal {
+  padding: 24px;
+  width: 480px;
+
+  .title {
+    display: flex;
+    justify-content: space-between;
+    padding-bottom: 16px;
+  }
+
+  .body{
+    padding-top: 4px;
+    padding-bottom: 4px;
+
+    .warning-text {
+      font-size: 14px;
+
+      .Text {
+        line-height: 20px;
+      }
+
+      .Text.top, .Text.middle {
+        margin-bottom: 20px;
+      }
+    }
+
+    .actions {
+      padding-top: 16px;
+      gap: 8px;
+      display: flex;
+      justify-content: flex-end;
+    }
+  }
+
+}
```

Comment:
1. prettier: file is not formatted correctly
2. Change name of the file to ArchiveThreadModal.scss

Reply:
Done!